docs/Features/XIncludeFeatures.md

```diff
@@ -0,0 +1,5 @@
+# XInclude Features
+
+## Validation
+
+XML validation with XInclude is supported, which can be enabled / disabled with the `xml.validation.xInclude.enabled` setting.
```

Comment:
Please add a screenshot with XInclude error.

Reply:
Please reference this page in https://github.com/redhat-developer/vscode-xml/blob/main/docs/Features.md

Reply:
Please reference this page in the README to promote that we have an XInclude support.